pkg/operator/ceph/cluster/osd/pod.go

```diff
@@ -152,18 +152,29 @@ func (c *Cluster) makeDeployment(nodeName string, devices []rookalpha.Device, se
 	if !osd.IsDirectory && osd.IsFileStore {
 		// filestore on a device requires indirection through the rook entrypoint so we can mount the image
 		sourcePath := path.Join("/dev/disk/by-partuuid", osd.DevicePartUUID)
-		args = append([]string{
+		mountPath := "/rook"
+		command = append([]string{
```

Comment:
It seems like the command for these containers was being used for both the command and the arguments. Do we want to separate them here to follow what the mon and mgr do?

Reply:
yes, it would be better to be consistent and separate the command from the args. I can't think of a good reason all the args were with the command other than laziness.